### PR TITLE
Contributing section

### PR DESCRIPTION
**Pull request by tertsdiepraam**
I've added a contributing section to the homepage. 

Also, I came up with a silly little slogan-like text:

> updated universal ultra-useful ultimate ubiquitous unix utilities

but it's in a separate commit so I can remove it if it's too silly. **EDIT: I REMOVED THE COMMIT**

I was inspired by the discussion on the coreutils repo about what the "u" stands for 😄 https://github.com/uutils/coreutils/discussions/5180

### CODE REVIEW

File: homepage/index.html [hunk 1]
```diff
@@ -302,6 +316,17 @@ <h2>Crates</h2>
...
           </ul>
+          <h2>Contributing</h2>
+          <p>You can help us out by:</p>
+          <ul>
+            <li>Contributing code</li>
+            <li>Contributing documentation</li>
+            <li>Reporting bugs (e.g. incompatibilities with GNU utilities)</li>
+            <li>Triaging bugs</li>
+            <li><a href="https://github.com/sponsors/uutils">Sponsoring uutils on GitHub</a></li>
+            <li>etc.</li>
```
Comment (cakebaker) [inline homepage/index.html:327]: I would either remove "etc" or replace it with the additional points implied by "etc", though I think you already mentioned the main points.

**Comment by cakebaker** [comment]
I think the slogan is too silly and I wouldn't understand it if I didn't read your answer in https://github.com/uutils/coreutils/discussions/5180 ;-)

**Comment by tertsdiepraam** [comment]
Fair enough :smile I'll remove it

**Review by cakebaker — APPROVED** [review]

**Comment by tertsdiepraam** [comment]
Alright, let's deploy this :)

**Pull request merged**